OutOfSchool/OutOfSchool.WebApi/Services/IWorkshopServicesCombiner.cs

```diff
@@ -18,12 +18,12 @@ public interface IWorkshopServicesCombiner : ICRUDService<WorkshopDTO>
         Task<SearchResult<WorkshopCard>> GetAll(OffsetFilter offsetFilter);
 
         /// <summary>
-        /// Get all workshops with the specified provider's Id.
+        /// Get all workshop cards with the specified provider's Id.
         /// </summary>
         /// <param name="id">Provider's key.</param>
-        /// <returns>A <see cref="Task{WorkshopDTO}"/> representing the result of the asynchronous operation.
-        /// The task result contains a <see cref="IEnumerable{WorkshopDTO}"/> that contains elements from the input sequence.</returns>
-        Task<IEnumerable<WorkshopDTO>> GetByProviderId(long id);
+        /// <returns>A <see cref="Task{WorkshopCard}"/> representing the result of the asynchronous operation.
+        /// The task result contains a <see cref="IEnumerable{WorkshopCard}"/> that contains elements from the input sequence.</returns>
+        Task<IEnumerable<WorkshopCard>> GetByProviderId(long id);
```

Comment:
Why we are returning `IEnumerable` instead of full type which we are already have with all it metadata ?